system-parachains/constants/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "system-parachains-constants"
+repository.workspace = true
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+
```

Comment:
Description please.

Reply:
Added in [6542acb](https://github.com/polkadot-fellows/runtimes/pull/115/commits/6542acb3e22f518882a150eac658f2618ae41f9e)